templates/content-single-ms_directory.php

```diff
@@ -724,6 +723,16 @@ function manager_industry( $manager_industry ) {
 											?>
 										</ul>
 									</div>
+
+									<?php
+									if ( isset( $url ) ) {
+										?>
+										<div class="AffiliateManagerCard__showProfile">
+                                            <a href="<?= esc_url( add_query_arg( array('directory_name' => get_the_title(), 'directory_url' => get_the_permalink() ), $url ) ); ?>" class="learn-more" title="<?= esc_html( $url_title ); ?>"><?php _e( 'Show profile', 'ms'); ?><?php echo QualityUnit\Extras\get_svg_image( 'arrow_small_right' ); // @codingStandardsIgnoreLine?></a>
```

Comment:
Please fix intendation of <a href part (too many spaces, use less intendation using tabs)

Reply:
fixed